code/modules/clothing/suits/jobs.dm

```diff
@@ -259,6 +280,30 @@
 	blood_overlay_type = "coat"
 	body_parts_covered = UPPER_TORSO|ARMS
 
+//Head of Security
+/obj/item/clothing/suit/mantle/armor
+	name = "armored shawl"
+	desc = "A reinforced shawl, worn by the Head of Security. Do you dare take up their mantle?"
+	icon_state = "hosmantle"
+	item_state = "hosmantle"
+	allowed = list(/obj/item/gun/energy, /obj/item/reagent_containers/spray/pepper, /obj/item/gun/projectile, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/flashlight/seclite, /obj/item/melee/classic_baton/telescopic, /obj/item/kitchen/knife/combat)
+	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0)
+	cold_protection = UPPER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT

```

Comment:
I should have been more clear. Because of inheritance, this is already defined; you don't need to define it again. What I meant was that the value should be `ARMOR_MIN_TEMP_PROTECT`